### ISSUE TITLE
Sending Remote packets to an F16v3

### ISSUE THREAD

**Issue by JonB256** [FalconChristmas/fpp#443]
Two things -

First: If you have two F16v3s in Remote mode and enter their IP addresses in the CVS line of Multisync, 
DO NOT put a space after the comma. If you do, FPPD will not start. Recommend either fix the parsing so that the space isn't critical or change the text to say "no space after the comma"

Second:  I'm upgraded to the b021 firmware on the F16v3 and Remote is no longer working (wired or wireless). This was working with b018. When I get a chance, I'll go back to b018 to see if that is the problem or if the problem is new.

**Comment by JonB256** [comment]
After a brief discussion with David Pitts, this is working.
Turns out that for F16v3 Remote mode to work (b021 firmware), that the FPP Master needs to use All Remotes for Multi-Sync.  Once I changed that back (because I was using that before when it worked), it is back to full function. 

As far as the CSV line, when you are in All Remotes mode, that line doesn't matter at all. Just leave it blank. (mine is now)

**Comment by dkulp** [comment]
Is that something he's fixing?   That's really really bad.

**Comment by dkulp** [comment]
I want to get rid of the broadcast for "All Remotes" so that will likely not be a good "long term" solution.

**Comment by dpitts** [comment]
It is actually an FPP/F16V3. Until the new discovery protocol Chris wrote is deployed in FPP and F16V3 there is no way to select a Unicast F16v3 in FPP via GUI.

**Comment by dkulp** [comment]
I guess I'm a bit confused.  There is currently an entry field to enter the IP addresses of "CSV MultiSync Remote"s.   Is that not for the F16V3?   Does the F16V3 use the binary sync packets and not the CSV?  If so, I can add an entry field for those as well.  That's not a big deal.   The other issue is why was it working with b018 and not b021?

**Comment by dpitts** [comment]
Yes. It uses the binary. At one time we were going to use CSV. The CSV is acutally not going to be used at all.

**Comment by JonB256** [comment]
Dan, the reason it "worked" in b018 was that I was using All Remotes at that time.
At about the same time b021 came out, you discussed the disadvantages of All Remotes on the forum.
I changed both at the same time, therefore masking the true failure cause.

**Comment by dkulp** [comment]
OK.  Thanks.  That's the info I need.  We'll change that field over to be regular sync packets instead of CSV.

**Comment by JonB256** [comment]
Dan, that change is working. In fact, since my F16v3 has both wired and wireless, I tried both (not at the same time; that's like crossing the streams in Ghostbusters). Both IPs worked.

The reason I checked is that the F16v3 has a page (Misc) to specify Wired or Wireless. With it set as Wired, it worked for both. That option may be intended for "transmitting" Remote packets from the the F16v3 in Master mode. I can't tell.  Thanks for the update. I'm out of All Remotes mode again.
